deploy/examples/object-multi-instance-test.yaml

```diff
@@ -0,0 +1,107 @@
+#################################################################################################################
+# Create an object store with settings for a test environment. Only a single OSD is required in this example.
+#  kubectl create -f object-multi-instance-test.yaml
+#
+# The example below provides minimalistic test configuration for multi-instance RGW deployment.
+# It defines CephObjectRealm, CephObjectZoneGroup, and CephObjectZone CRs responsible for storage pools configuration.
+# Then, three separate CephObjectStore resources are created.
+# These stores refer to the same CephObjectZone, and therefore, to the same storage pools, representing the same ObjectStorage data.
+# The CephObjectStore resources are configured to host different RGW APIs: S3, SWIFT, and Admin-ops.
+# The configuration of the CephObjectStore can be further extended to include any number of RGW deployments backed by the same data.
+# Deployments can be customized to host different APIs, allocate different resources, use different domains,
+# and apply any other options available in the CephObjectStore CRD
+#################################################################################################################
+apiVersion: ceph.rook.io/v1
+kind: CephObjectRealm
+metadata:
+  name: my-store
+  namespace: rook-ceph # namespace:cluster
+---
+apiVersion: ceph.rook.io/v1
+kind: CephObjectZoneGroup
+metadata:
+  name: my-store
+  namespace: rook-ceph # namespace:cluster
+spec:
+  realm: my-store
+---
+apiVersion: ceph.rook.io/v1
+kind: CephObjectZone
+metadata:
+  name: my-store
+  namespace: rook-ceph # namespace:cluster
+spec:
+  zoneGroup: my-store
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 1
+      requireSafeReplicaSize: false
+  dataPool:
+    failureDomain: host
+    replicated:
+      size: 1
+      requireSafeReplicaSize: false
+    parameters:
+      compression_mode: none
+  # Alternatively, configure pool placements with pre-existing pools here.
+  # More details about Pool placements and storage classes:
+  # https://rook.io/docs/rook/latest-release/Storage-Configuration/Object-Storage-RGW/object-storage/#create-local-object-stores-with-pool-placements
+  # sharedPools:
+  #   poolPlacements:
+  #   ...
+---
+# RGW instance to host admin-ops API only
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: store-admin
+  namespace: rook-ceph # namespace:cluster
+spec:
+  gateway:
+    port: 80
+    instances: 1
+  zone:
+    name: my-store
+  protocols:
+    enableAPIs: ["admin"]
+---
+# RGW instance to host S3 API only
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: store-s3
+  namespace: rook-ceph # namespace:cluster
+spec:
+  gateway:
+    port: 80
+    instances: 1
+  zone:
+    name: my-store
+  protocols:
+    enableAPIs:
+      - s3
+      - s3website
+      - sts
+      - iam
```

Comment:
I have been wondering about this for a while: is `iam` permission something users should have, or admin? This may be a matter of opinion and user intent as well. No change requested here, but something I'm curious about.

Reply:
Good question. I hope that it will work both ways but ceph docs says:

> This REST API is served by the same HTTP endpoint as the Ceph Object Gateway S3 API.